web/testing/commonHeader.js

```diff
@@ -67,7 +77,7 @@
         id:'lo',name:'Lao',region:'Asia',
         font:{family:'LaoWeb',source:['../font/saysettha_web.ttf','../font/saysettha_web.woff','../font/saysettha_web.eot']}
         },
```

Comment:
The saysettha_web font reference here has been broken (along with the corresponding one in /samples/)